bears/general/InvalidLinkBear.py

```diff
@@ -79,14 +81,15 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             match = regex.search(line)
             if match:
                 link = match.group()
-                if not link_ignore_regex.search(link):
+                if not (link_ignore_regex.search(link) or
+                        fnmatch(link, links_to_ignore)):
                     if link.startswith(('hg+', 'bzr+', 'git+', 'svn+')):
                         link = InvalidLinkBear.parse_pip_vcs_url(link)
                     code = InvalidLinkBear.get_status_code(link, timeout)
                     yield line_number + 1, link, code
 
     @deprecate_settings(link_ignore_regex='ignore_regex')
-    def run(self, filename, file,
+    def run(self, filename, file, links_to_ignore: typed_list(str)='',
```

Comment:
@jayvdb I've made changes as you suggested (Changed ```links_to_ignore``` to ```link_ignore_list```). However I think using ```''``` instead of ```None``` is correct because ```fnmatch``` function expects a glob string which can be ```''``` by default.